lir/config/visualization.py

```diff
@@ -15,15 +17,35 @@ def parse_visualizations(config: ContextAwareDict, output_path: Path) -> list[Ca
        - labels: np.ndarray, a list of labels corresponding to the lrs
     """
     visualization_functions = []
-    for visualization_type in config:
+    for item in config:
+        # support both simple list entries and dict entries
+        # (e.g. `- pav: {h1_color: yellow, h2_color: green}` or just `- pav`)
+        if isinstance(item, str):
+            visualization_type = item
+            module_cfg = None
+        elif isinstance(item, Mapping):
+            if len(item) != 1:
+                raise YamlParseError(config.context, f"invalid visualization entry: {item}")
```

Comment:
Deze error krijg je als je een streepje vergeet:
```yaml
- pav:
    ...
  ece:
    ...
```

Dan kan deze error misschien wat explicieter. Of: accepteer het als geldige input voor twee visualisaties.